### ISSUE TITLE
How you publish a premium Extension as open files

### ISSUE THREAD

**Issue by kvvaradha** [ameedev/ExtendedHrm#1]
Hello,

How do you publish my modules over internet?.  

Who gave you the rights to publish it. ?

**Comment by ameedev** [comment]
Since this module was developed from Frontaccounting which follows the GPL v3 licence no one has the right to ban others from spreading it.

**Comment by kvvaradha** [comment]
you are mistaken it.  Ofcouse FA(frontaccounting ) is open source. But the modules which i developed is my code.you can license it under GPL. I didnt provbide GPL license and showed it.


Who gave you rights to spread my Program  ?


And let me ask you a question. Wordpress is a open source. Will you find the WordPress themes and plugins from Premium sellers and put it here.  ?

Is it legal to Spread like this.?

**Comment by kvvaradha** [comment]
As you are developer,? 

If i release your program online without getting proper rights from you , Will you accept it ?

You are not supposed to share my programs online without My license.

**Comment by kvvaradha** [comment]
answer me man ?

**Comment by ameedev** [comment]
Artical 5 section c in the GNU General Public License states:

> You must license the entire work, as a whole, under this License to anyone who comes into possession of a copy. This License will therefore apply, along with any applicable section 7 additional terms, to the whole of the work, and all its parts, regardless of how they are packaged. This License gives no permission to license the work in any other way, but it does not invalidate such permission if you have separately received it.

You yourself have violated this provision without including GPL license on your product

**Comment by kvvaradha** [comment]
Hey man, you are unnecessarily boiling my work. Since you are showing the GNU. you should remember one thing. Its not specified anything about the modules or add-on developed to it.  you are stealing my work and releasing online without my concern. Is it legal from your view. ? 

See the FA is GPL. But its does not mean that i have to give my works under GPL. 

Simply answer me. Wordpress is also GPL . So will you steal the premium themes and plugins  from themeforest and release it like this.  ?

**Comment by kvvaradha** [comment]
You mean to say, Those who are selling themes and plugins of Wordpress is belongs to be violation of terms of GPL. And all has to be open sourced. to provide it to others, right?

Is that you mean.?

I dont want to argue with you. tell. how did you get these files.? 

And who gave you the rights to show it online without checking with me. Even if you think it as open source, you must checked with me before spreading it here.  

This way you are not followed any steps to question me now.  

answer me. how did you get files of mine.  
I know @Joe and @Janusz, They already saw my website and checked my modules.  They never asked me to make it open source,  Who are you to license my efforts under GPL. ? you have not even be in FA forum to support open source.  Than why are you doing my works online. 

You know how much efforts i have invested to develop this module. you are entirely eating my efforts.

**Comment by ameedev** [comment]
In case of WordPress plugins you don't have to make them open source but
If someone buy your plugins then gives them away for free there is nothing you can do.
That is the main idea of GPL license.

**Comment by kvvaradha** [comment]
Thats not GPL License idea.  
May i know who gave you the program to use it free.  Dont you think its life of a developers effort behind it to show it to online without my knowledge. 

If you are true to your heart.  You shouldn't keep my files and effort as open source.  if you wish not to hurt me. remove this remove and keep my files secure for me.

**Comment by kvvaradha** [comment]
are you checking it ?

**Comment by ameedev** [comment]
I'm evaluating the possibility of improve this module
I will remove if can't find it's development potential
Let's see, I guess the result would be available this week.

Anyway, thank you for contributing to the community that I also benefit from

**Comment by kvvaradha** [comment]
This is not contributing to the community.  You are thief, steal my code and you are going to make business with it.  You are a big sinner man.  you will face the issues soon. Good luck.

**Comment by kvvaradha** [comment]
@ameedev  So you want me to do any legal steps aganist you.

**Comment by ameedev** [comment]
You should do any steps that follow the rules of the community we are in.
If there is any legal evidence indicates I'm wrong, I will delete this repository right away.
